### DISCUSSION TITLE
Bad Extra Network

### DISCUSSION THREAD

**Discussion by swordworksAI** [Nevysha/Cozy-Nest#169] [General]
The new Extra network is very difficult to use, the thumbnails are not recognized, and the icon proportions are incorrect. I must return to the previous version.

新的Extra网络非常不好用，缩略图不认识，错误的图标比例，我必须退回以前的版本。

**Comment by Nevysha** [comment]
https://github.com/Nevysha/Cozy-Nest/issues/new?assignees=&labels=&projects=&template=bug_report.yml&title=%5BIssue%5D%3A+
